File: jcarrollonlinev4.backend/Models/MicroPosts/MicroPostFeedModel.cs
namespace jcarrollonlinev4.backend.Models.Micropost
{
    public class MicropostFeedModel : MicropostModelBase
    {
        public MicropostFeedModel()
        {
            MicroPostFeedItems = new List<MicropostFeedItemModel>();
        }
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public ICollection<MicropostFeedItemModel> MicroPostFeedItems { get; set; }
        //public IPagedList<MicroPostFeedItemModel> OnePageOfMicroPosts { get; set; }
    }
}

File: jcarrollonlinev4.frontend/src/models/MicroPosts/MicroPostFeedModel.ts
import { MicropostFeedItemModel } from "./MicroPostFeedItemModel";
import { MicropostModelBase } from "./MicroPostModelBase";

export interface MicropostFeedModel extends MicropostModelBase {
  MicroPostFeedItems: MicropostFeedItemModel[];
  OnePageOfMicroPosts: MicropostFeedItemModel[];
}
